src/status_im/wallet/accounts/core.cljs

```diff
@@ -265,7 +265,7 @@
                                  :on-success #()}]
                :db (-> db
                        (assoc :multiaccount/accounts new-accounts)
-                       (assoc-in [:wallet :accounts deleted-address] nil))}
+                        (update-in [:wallet :accounts] dissoc deleted-address))}
```

Comment:
please remove an extra whitespace before `(update`, that's the reason why builds fail